src/main/java/marquez/db/JobVersionDao.java

```diff
@@ -74,7 +75,7 @@ default void insertAndUpdate(JobVersionRow row) {
   void update(UUID rowUuid, Instant updatedAt, UUID latestRunUuid);
 
   @SqlQuery(
-      "SELECT j.uuid AS job_uuid, j.namespace_uuid, jv.*, "
+      "SELECT j.uuid AS job_uuid, j.namespace_uuid, jv.*, jc.uuid AS job_context_uuid, jc.context, "
```

Comment:
it's bit risky to return  explicit columns mixed with * of some other tables. to maintain in the long run,  we usually specify all columns explicitly.  so when project goes very large, it's easy to find out by searching for the column name.  But we probably can leave this kind of refactoring work into future.